crypto.go

```diff
@@ -91,6 +91,15 @@ type PublicKey struct {
 	ed25519PublicKey *_Ed25519PublicKey
 }
 
+/**
+ *  SDK needs to provide  a way to set an unusable key such as an Ed25519 all-zeros
+ *  key, since it is (presumably) impossible to find the 32-byte string whose SHA-512 hash begins with 32 bytes
+ *  of zeros. We recommend using all-zeros to clearly advertise any unusable keys.
+ */
+func UnusableKey() (PublicKey, error) {
```

Comment:
nit: Would just avoid using unusable as the key name it might bring confusion. Instead lets leave it zeroKey or something similar.

Reply:
Renamed